MarkBind/markbind
# Add documentation on indentation support for components

**What is the purpose of this pull request?**




**Overview of changes:**
Fixes https://github.com/MarkBind/markbind/issues/1015

As mentioned in the above issue, indenting components might be a common use case. Let's include a section in the tips and tricks to inform users on how to use bootstrap classes to indent components.

Changes include:
- enable pageNav on the tips and tricks page for easier navigation
- add in "indent component section"
- include the above section under "using components/presentation" for more visibility
- a minor update on the formatting content page to make "tips and tricks" visible in pageNav

**Anything you'd like to highlight / discuss:**
I see that there are "add-class" for adding classes. I used "class" directly as it is shorter.

**Testing instructions:**
See preview:
- [indent components](https://deploy-preview-1719--markbind-master.netlify.app/userguide/tipsandtricks#indent-components)
- [presentation](https://deploy-preview-1719--markbind-master.netlify.app/userguide/components/presentation#relevant-tips-and-amp-tricks)
- [minor update on heading for the formatting content page](https://deploy-preview-1719--markbind-master.netlify.app/userguide/formattingcontents#relevant-tips-and-amp-tricks)


**Proposed commit message: (wrap lines at 72 characters)**
Add documentation on indentation support for components

Users may need to indent components to align with nearby
content.

Let's add in a section in 'tips and tricks' to show examples
of how to indent components with bootstrap utility classes.



---

**Checklist:** :ballot_box_with_check:



   - a minor update on the "formatting content" page to make "tip and tricks" visible in pageNav
